testCases/testAll.sh

```diff
@@ -62,17 +62,12 @@ printf "\e[0m";
 
     printf '%s\n' "${testCase/%?}";
 
-    printf "\e[96m"
-    printf '%s\n' "--expected:"
-    printf "\e[92m"
-    echo "$expected"
-
-    printf "\e[96m"
-    printf '%s\n' "--actual:"
-
-    printf "\e[35m"
-    echo "$actual"
-    printf "\e[0m"
+    diff \
```

Comment:
try failing one test, now it show nice colorfull diff! 😍

Reply:
Great. Good job!

Reply:
The test runner it's becoming a very interesting project by itself. You should consider publish it.

Reply:
once it is done i will implement it in mishess.sh, or even create repo itself from that? hm :) but right now, versiontag is playground where we want to go :)

(lol, if this tester will be repo of its own, how should we test it? :D :D)